viewers/nerf_viewer/bridge_server/viewer_state.py

```diff
@@ -0,0 +1,92 @@
+import time
+import base64
+import pickle
+import umsgpack
+from typing import Tuple
+import numpy as np
+import cv2 as cv
```

Comment:
What about not as `cv`? We use `cv2` directly in other modules inside XRPrimer.

Reply:
Changed `cv` to `cv2`. The OpenCV is used to generate fake images for debug purpose, which should be further replaced by the nerf backend. Hence, the dependency will be removed once the nerf backend is ready.